src/index.ts

```diff
@@ -0,0 +1,61 @@
+import express from 'express'
+import promBundle from 'express-prom-bundle'
+import { Metrics } from '@eth-optimism/common-ts'
+import * as dotenv from 'dotenv'
+
+import { ReplicaMetrics } from './types'
+import { runSyncCheck } from './sync-check'
+
+
+/**
+ * Helper functions
+ */
+
+const initMetrics = (app: express.Express): ReplicaMetrics => {
+  const metrics = new Metrics({
+    prefix: 'replica_health',
```

Comment:
Please remove this prefix as it's being added to all the standard node metrics and will cause problems reusing generic node dashboards.